src/realm/util/string_buffer.cpp

```diff
@@ -23,21 +23,19 @@
 
 #include <realm/util/safe_int_ops.hpp>
 #include <realm/util/string_buffer.hpp>
+#include <realm/utilities.hpp>
 
 using namespace realm;
 using namespace realm::util;
 
 
-char StringBuffer::m_zero = 0;
-
-
```

Comment:
VC could not resolve this private member in UWP mode, for some reason; gave compiler error